SECP-service/src/main/resources/assets/app/index.html

```diff
@@ -1,26 +1,28 @@
 <!doctype html>
 <html lang="en" ng-app="SECP">
-    <head>
-        <base href="/">
-        <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>SECP</title>
-        <link rel="shortcut icon" type="image/ico" href="img/favicon.ico"/>
-        <link href="lib/bootstrap-css/css/bootstrap.css" rel="stylesheet">
-        <link href="lib/jquery-ui/themes/smoothness/jquery-ui.css" rel="stylesheet"/>
-        <link href="lib/components-font-awesome/css/font-awesome.min.css" rel="stylesheet"/>
-  	    <link href="lib/simple-line-icons/css/simple-line-icons.css" rel="stylesheet">
- 	    <link href="img/home/device-mockups/device-mockups.css" rel="stylesheet">
- 	    <link href="lib/simple-line-icons/css/simple-line-icons.css" rel="stylesheet">
-	    <link href="img/home/device-mockups/device-mockups.css" rel="stylesheet">
-        <link href="css/app.css" rel="stylesheet">
-        <link href="lib/sweetalert2/dist/sweetalert2.css" rel="stylesheet">
-        <link href="lib/select2/dist/css/select2.css" rel="stylesheet">
-        <link href="lib/datatables.net-jqui/css/dataTables.jqueryui.min.css" rel="stylesheet">
-        <link href="lib/datatables.net-select-jqui/css/select.jqueryui.css" rel="stylesheet">
-    </head>
-    <body>
+
```

Comment:
Revert this whole file, there is duplicates and it doesn't seem like you added anything.

Reply:
I had to add my controllers and delete duplicates that were somehow there.

Reply:
There were'nt any duplicates originally, you actually added some duplicates.

Reply:
damn thats messed. but i still had to add my controllers. I tried removing all the duplicates let me know if theres more